src/test/java/wellnus/tokenize/ReflectionTokenizerTest.java

```diff
@@ -0,0 +1,54 @@
+package wellnus.tokenize;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import wellnus.exception.TokenizerException;
+import wellnus.storage.ReflectionTokenizer;
+
+public class ReflectionTokenizerTest {
+    private static final int INDEX_ZERO = 0;
+    private static final int INDEX_ONE = 1;
+    @Test
+    void tokenizeReflect_checkOutput_success() throws TokenizerException {
+        ArrayList<Set<Integer>> indexToTokenize = new ArrayList<>();
+        Set<Integer> likeTestIndexes = new HashSet<>();
+        Set<Integer> prefTestIndexes = new HashSet<>();
+        likeTestIndexes.add(1);
+        likeTestIndexes.add(2);
+        prefTestIndexes.add(3);
+        prefTestIndexes.add(4);
+        indexToTokenize.add(likeTestIndexes);
+        indexToTokenize.add(prefTestIndexes);
+        String expectedTokenizedLike = "like 1,2";
+        String expectedTokenizedPref = "pref 3,4";
+        ArrayList<String> expectedTokenizedIndex = new ArrayList<>();
+        expectedTokenizedIndex.add(expectedTokenizedLike);
+        expectedTokenizedIndex.add(expectedTokenizedPref);
+        ReflectionTokenizer reflectionTokenizer = new ReflectionTokenizer();
+        ArrayList<String> actualTokenizedIndex = reflectionTokenizer.tokenize(indexToTokenize);
+        Assertions.assertEquals(expectedTokenizedIndex, actualTokenizedIndex);
+    }
+    @Test
+    void detokenizeReflect_checkOutput_success() throws TokenizerException {
+        Set expectedDetokenizedLike = new HashSet<>();
```

Comment:
1. Don't forget the angle brackets for `Set`, your IDE should have flagged this as a warning
2. I recommend something like `expectedDetokenizedLikes` instead, this is a plural data structure